### PR TITLE
[IMP] spreadsheet: Suport deterministic action for drilldown

### PR DESCRIPTION
**Pull request by rrahir**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/spreadsheet/static/src/list/list_actions.js [hunk 1]
```diff
@@ -14,19 +15,19 @@ export const SEE_RECORD_LIST = async (position, env) => {
...
     }
-    await env.services.action.doAction({
+    await navigateTo(env, actionXmlId, {
         type: "ir.actions.act_window",
         res_model: model,
         res_id: recordId,
         views: [[false, "form"]],
         view_mode: "form",
-    });
+    }, { viewType: "form" });
```
Comment (LucasLefevre) [inline addons/spreadsheet/static/src/list/list_actions.js:30]: What's the difference between `view_mode` and `viewType` again ?

**Review by LucasLefevre — COMMENTED** [review]
it's looking good :)
An old limitation gone :disguised_face:

File: addons/spreadsheet/static/src/list/list_actions.js (hunk 1 above)
Comment (rrahir) [reply addons/spreadsheet/static/src/list/list_actions.js:30]: view mode is deprecated

**Review by LucasLefevre — APPROVED** [review]
robodoo r+ rebase-ff

**Pull request closed without merging**